backend/.env

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb://127.0.0.1:27017/PartsUnlimited
```

Comment:
.env files is usually not checked in to repo because it might contains secrets. It's better to check in template file and explain in the readme what it should contains